Dockerfile

```diff
@@ -16,12 +16,12 @@ RUN set -x && \
     # Clean-up
     apk del .deps
 
+ARG DOCKER_COMPOSE_VERSION
+ENV DOCKER_COMPOSE_VERSION=${DOCKER_COMPOSE_VERSION:-"1.10.0"}
```

Comment:
I'd inline the ENV to avoid polluting the final environment. Also please use latest by default. If you have a need for an earlier version, I'd prefer to create tags, but for that I'd like to understand that there is a need for it.